lib/matplotlib/animation.py

```diff
@@ -1124,7 +1124,8 @@ class to use, such as 'ffmpeg'. If ``None``, defaults to
                                          extra_args=extra_args,
                                          metadata=metadata)
             else:
-                _log.warning("MovieWriter %s unavailable.", writer)
+                _log.warning("MovieWriter {} unavailable. Using {} \
```

Comment:
Can this be "Trying to use" instead, because the following code block can still raise an error.

Reply:
Changed.